### DISCUSSION TITLE
Difference between other Game Panels

### DISCUSSION THREAD

**Discussion by ghost** [et-nik/gameap#53] [Q&A] [answered]
Hello, what's the difference between other game managers.

I'm currently using OpenGamePanel, and I have plan to switch on this one. 

Questions:
1. Is this better than OGP?
2. Consumes less RAM and space?
3. CPU friendly? I mean does it consumes less CPU usage?

That's all, thanks.

**Comment by et-nik** [comment, marked as answer]
Hello. There are users who migrate from OGP to GameAP. 
I didn't do a performance comparison, but it is good idea.

1. Users reported yes (See official discord channel, and telegram channel).
2. Mostly yes.
3. GameAP design focuses on performance, less CPU usage and flexible functionality.
